opsdroid/connector/matrix/create_events.py

```diff
@@ -30,7 +30,8 @@ class MatrixEventCreator(events.EventCreator):
 
     async def create_event_from_eventid(self, eventid, roomid):
         """Return an ``Event`` based on an event id in a room."""
-        event_json = await self.connector.connection.get_event_in_room(roomid, eventid)
+        room_context = await self.connector.connection.room_context(roomid, eventid, 1)
+        event_json = room_context.event
```

Comment:
`room_context.event` returns a `RoomMessageText` instead of a dict
```suggestion
        event_json = room_context.event.source
```

Also [down here](https://github.com/opsdroid/opsdroid/pull/1418/files#diff-9d595b55ddf1583136a938382e129321R122) in `_file_kwargs()` the function is not part of nio, instead the url can be retrieved like so 
```
url = event["content"]["url"]
```